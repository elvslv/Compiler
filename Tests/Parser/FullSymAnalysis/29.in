var i, j: integer;
function func(i : integer): integer;
begin
       result := i;
end;
begin
        for i := 1 to func(10) do
                j := 5;
end.