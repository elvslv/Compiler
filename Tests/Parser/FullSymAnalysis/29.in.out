var I : INTEGER;
var J : INTEGER;

function FUNC(var I : INTEGER;): INTEGER;

begin         
|             
|___:=        
|   |         
|   |___RESULT
|   |         
|   |___I     
|             
end           


begin            
|                
|___for          
|   |            
|   |___:=       
|   |   |        
|   |   |___I    
|   |   |        
|   |   |___1    
|   |            
|   to           
|   |            
|   |___()       
|   |   |        
|   |   |___FUNC 
|   |   |        
|   |   |___10   
|   |            
|   do           
|   |            
|   |___begin    
|       |        
|       |___:=   
|       |   |    
|       |   |___J
|       |   |    
|       |   |___5
|       |        
|       end      
|                
end              
